dmutils/filters.py

```diff
@@ -66,3 +66,26 @@ def capitalize_first(maybe_text):
         return [capitalize_first(item) for item in maybe_text]
 
     return maybe_text
+
+# find repeated sequences of '\r\n\', optionally separated by a space character
+_multiple_newlines_re = re.compile(r'(\r\n ?){2,}')
+# find \r\n sequences
+_single_newline_re = re.compile(r'(\r\n)')
+
+
+@evalcontextfilter
+def preserve_line_breaks(eval_ctx, value, question_type=None):
+    if question_type and question_type != 'textbox_large':
+        return value
+
+    value = '{}'.format(escape(value))
```

Comment:
Probably good to leave a comment about what this is for.  It's not really obvious that it's turning (potential) Markdown objects back into strings.